src/redux/reducers/movies.js

```diff
@@ -0,0 +1,48 @@
+import {
+    SORTING_BY_DATE,
+    SET_FULL_LIST_ACTION,
+    SET_SORTING_TYPE_ACTION
+} from "../../constants";
+
+const transformMovieData = (data) => ({
```

Comment:
function transformMovieData already exist in actions/index.js file. Please remove duplicate

Reply:
these functions provide different transforms